src/components/calcite-combobox/calcite-combobox.tsx

```diff
@@ -891,12 +897,11 @@ export class CalciteCombobox implements LabelableComponent, FormComponent {
       const label = selectionMode !== "ancestors" ? item.textLabel : pathLabel.join(" / ");
       return (
         <calcite-chip
-          aria-label={label}
           class={chipClasses}
           dismissLabel={"remove tag"}
```

Comment:
@jcfranco @benelan Need an issue for this string. Do we need/want a issue template for localization? Or would this be categorized as a bug? or feat cus it needs a new prop?

Reply:
I'd call that a bug since it is a hardcoded string. And then just link the doc site for the repro sample. I don't think we need a new issue template for localization, we can just use the existing labels. I'll make the issue for this.

Reply:
I think its more of a feature. Nothing is broken, we are just adding the ability to set it via a property.